includes/class-accredible-learndash-admin-scripts.php

```diff
@@ -66,11 +66,24 @@ public static function load_page_ajax() {
 					)
 				);
 			}
+		}
+
+		/**
+		 * Enqueues ajax scripts for pages.
+		 */
+		public static function load_page_scripts() {
```

Comment:
I like creating this new function to differentiate it from `load_page_ajax`. Nice 👍